estudios-de-fonetica-experimental.csl

```diff
@@ -0,0 +1,317 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title xml:lang="esp">Estudios de fonética experimental</title>
+    <title-short>EFE</title-short>
+    <id>http://www.zotero.org/styles/estudios-de-fonetica-experimental</id>
+    <link href="http://www.zotero.org/styles/estudios-de-fonetica-experimental" rel="self"></link>
+    <link href="http://www.zotero.org/styles/harvard-university-of-greenwich" rel="template"/>
+    <link href="http://stel.ub.edu/labfon/en/content/publication-estudios-de-fonetica-experimental#Normes" rel="documentation"/>
+    <author>
+      <name>Wendy Elvira-García</name>
+      <uri>https://www.wendyelvira.ga</uri>
+    </author>
+    <contributor>
```

Comment:
I guess you can remove yourself as a contributor since you're the author already.
The other contributor actually contributed?

Reply:
Ok, I'll remove myself. The second contributor is there as the contact with the editorial board of the journal. Whoever is managing that e-mail will keep the maintenance of the style.

Reply:
> The second contributor is there as the contact with the editorial board of the journal. Whoever is managing that e-mail will keep the maintenance of the style.

So you've been in contact with the editorial board about this style?

Reply:
Yes, I am part of it